prime-router/src/main/kotlin/common/HttpClientUtils.kt

```diff
@@ -312,7 +330,7 @@ class HttpClientUtils {
             httpClient: HttpClient? = null,
         ): HttpResponse {
             return runBlocking {
-                (httpClient ?: createDefaultHttpClient(accessToken)).submitForm(
+                (httpClient ?: getDefaultHttpClient()).submitForm(
```

Comment:
Is this pattern still required given we have a singleton http client now? Or are there still some specialized clients sprinkled around the codebase?

Reply:
I don't think there are callers supplying their own clients - but - I'm loathe to make too many changes until we dig in and refactor all of this (including the sister class HttpUtils) into one, unified, set of http utilities.

Reply:
ok in that case I'd suggest just keeping that in mind when refactoring.